src/program/snabb_lwaftr/bench/bench.lua

```diff
@@ -45,5 +45,5 @@ function run(args)
    config.link(c, "statisticsv6.output -> sinkv6.input")
 
    app.configure(c)
-   app.main({})
+   app.main({duration=10})
```

Comment:
Nit: Ideally I would prefer to have a duration flag, like other scripts, with a default to 10.